src/parsable.rs

```diff
@@ -429,7 +429,7 @@ impl NameTracker {
         // Check if there are any unresolved forward label references.
         for (id, op) in label_scope {
             if matches!(op, LabelRef::ForwardRef(_)) {
-                input_err!(loc.clone(), UnresolvedReference(id.clone()))?
+                return input_err!(loc.clone(), UnresolvedReference(id.clone()))?;
```

Comment:
Why this change?

Reply:
Ooops, I wanted to clean that up. I had a conflict with the commit introducing locations when rebasing